src/loader.js

```diff
@@ -362,7 +362,7 @@ function setupModuleLoader(window) {
           component: function(name, options) {
             function factory($injector) {
               function makeInjectable(fn) {
-                if (angular.isFunction(fn)) {
+                if (isFunction(fn) || isArray(fn)) {
```

Comment:
`isFunction` will work because of https://github.com/angular/angular.js/blob/master/src/loader.prefix#L8 `isArray` will not work